### ISSUE TITLE
toggle battery_recharge_when_cheap_tibber_price

### ISSUE THREAD

**Issue by vistree** [martinknechtel/sungrow-tibber-home-assistant#2]
He, really nice automation!! I try the simulate one right now - and it seems to work great.
If I want to use the extended automation - am I right that I need to create the toggle "battery_recharge_when_cheap_tibber_price" on my own? Or is it created automatically when creating the automation?

**Comment by martinknechtel** [comment]
Thanks @vistree ! Exactly, you create this toggle via ```yourmachine:8123/config/helpers``` as described on https://www.home-assistant.io/integrations/input_boolean/. Would it help if I add this to the documentation?

**Comment by vistree** [comment]
Yes, I think one sentence would be helpful for new HA users like me ;-)

**Comment by martinknechtel** [comment]
Thanks @vistree! Solved with https://github.com/martinknechtel/sungrow-tibber-home-assistant/commit/2fc1e347a612ccbb5826c408bd84c512f3185fdc
